MailCore/Cache/Actions/Action.swift

```diff
@@ -0,0 +1,90 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import MailResources
+import SwiftUI
+
+public struct Action: Identifiable, Hashable, Equatable {
+    public let id: String
+    public let title: String
+    public let shortTitle: String?
+    public let iconName: String
```

Comment:
Why do we need `iconName` and `tintColorName` ?

Reply:
It was easier to do it like that to make Action automatically conform to hashable etc. And Image is not hashable